qhub/keycloak.py

```diff
@@ -2,14 +2,15 @@
 import os
 
 import keycloak
+import rich
 
 from .schema import verify
 from .utils import load_yaml
 
 logger = logging.getLogger(__name__)
 
 
-def do_keycloak(config_filename, *args):
+def do_keycloak(config_filename, add_user, listusers):
```

Comment:
This change is the reason the Kubernetes CI test is failing. Let's see if we can keep the original function while also making this work for our Typer CLI as well. This might just mean updating how we pass the above values to the function. A more robust solution might involve creating two subcommands, `add-user` and `listusers`.